guides/views.md

```diff
@@ -94,7 +95,9 @@ Now we can reload the page and view source to see what we have.
 <a href="/">Link back to this page</a>
 ```
 
-Great, `page_path/2` evaluated to `/` as we would expect, and we didn't need to qualify it with `Phoenix.View`.
+Great, `Routes.page_path/2` evaluated to `/` as we would expect, we just had to use the alias set in `Phoenix.View`.
+
+If you happen to need access to the path helpers outside views, controllers or templates, you can either call them by the full qualified name, `HelloWeb.Router.Helpers.*_path()` or alias it yourself in the calling module, by defining `alias HelloWeb.Router.Helpers, as: Routes` in the module you want to use, and then calling `Routes.*_path()`, where `Hello*` is the name of your actual application.
```

Comment:
Let's change this to use a concrete path, as `*_path` could be confusing. I'd use `page_path`.

I think we can also get rid of `, where `Hello*` is the name of your actual application.` because we are using the `HelloWeb` application throughout.

Reply:
Agree, it actually doesn't make much sense as we're speaking of specific cases.